wallet/how-to/manage-networks/use-multichain.md

```diff
@@ -0,0 +1,245 @@
+      method: msg.data.params.notification.method,
+      subscription: msg.data.params.notification.params.subscription,
+      number: msg.data.params.notification.params.result.number
+    })
+    return;
+  }
+  console.log(msg.data)
+})
+```
+
+### 2. Manage multichain connections
+
+To interact with multiple networks simultaneously, you'll create and manage
+[CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) multichain connections
+with MetaMask.
+
+#### 2.1. Check existing connections
+
+Before creating a new connection, check if one already exists using the
+[`wallet_getSession`](../../reference/multichain-api.md#wallet_getsession) method.
+For example:
+
+```javascript
+extensionPort.postMessage({
+  type: "caip-348",
+  data: {
+    jsonrpc: "2.0",
+      method: "wallet_getSession",
+      params: {}
+  }
+});
+```
+
+If the result returns an empty `sessionScopes` parameter, then a multichain connection is not active
+and you must create a new connection.
+
+#### 2.2. Create a new connection
+
+Create a new connection using the [`wallet_createSession`](../../reference/multichain-api.md#wallet_createsession) method.
+Specify which chains and methods your dapp needs access to, using the `optionalScopes` parameter.
+For example:
+
+```javascript
+extensionPort.postMessage({
+  type: "caip-348",
+  data: {
+    jsonrpc: "2.0",
+    method: "wallet_createSession",
+    params: {
+      optionalScopes: {
+        "wallet:eip155": { // General Ethereum wallet functions
+          methods: ["wallet_addEthereumChain"],
+          notifications: [],
+          accounts: []
+        },
+        "eip155:1": { // Ethereum Mainnet
+          methods: [
+            "personal_sign",
+            "eth_blockNumber",
+            "eth_gasPrice",
+            "eth_getBalance",
+            "eth_getTransactionCount",
+            "eth_sendTransaction",
+            "eth_subscribe"
+          ],
+          notifications: ["eth_subscription"],
+          accounts: []
+        },
+        "eip155:59141": { // Linea Sepolia
+          methods: [
+            "personal_sign",
+            "eth_blockNumber",
+            "eth_gasPrice",
+            "eth_getBalance",
+            "eth_getTransactionCount",
+            "eth_sendTransaction", 
+            "eth_subscribe"
+          ],
+          notifications: ["eth_subscription"],
+          accounts: []
+        }
+      }
+    }
+  }
+});
+```
+
+In `optionalScopes`:
+
+- Request access to networks that your dapp intends to interact with.
+  If a requested network is not configured by the MetaMask user, you might need to
+  [add the network](add-network.md).
+- For each network, request access to [Wallet API methods](../../reference/json-rpc-methods/index.md)
+  that your dapp expects to call at any time.
+  The methods listed in the `sessionScope` of each Multichain API response indicate which wallet
+  capabilities your dapp can use during the connection.
```

Comment:
session vs connection, what is the better term? 🤔

Reply:
This was raised by @vandan in a previous comment, so we switched to "connection" to see how it flows. I'm OK with either term.
> The CAIP standards extensively use the term "session", which insinuates a transient connection. MetaMask predominantly establishes persistent connections so we could consider using alternative terminology such as "connection". But I'm not sure if it's more confusing to diverge from the names used in the Multichain API methods.